aptos-move/framework/aptos-framework/sources/configs/Stake.move

```diff
@@ -619,26 +674,35 @@ module AptosFramework::Stake {
     }
 
     /// Validate that the lockup time is at least more than the minimum required.
-    fun validate_lockup_time(locked_until_secs: u64) {
+    fun validate_lockup_time(locked_until_secs: u64, validator_set_config: &ValidatorSetConfiguration) {
         let current_time = Timestamp::now_seconds();
-        assert!(current_time + MINIMUM_LOCK_UP_SECS <= locked_until_secs, Errors::invalid_argument(ELOCK_TIME_TOO_SHORT));
-        assert!(locked_until_secs <= current_time + MAXIMUM_LOCK_UP_SECS, Errors::invalid_argument(ELOCK_TIME_TOO_LONG));
+        assert!(
+            current_time + validator_set_config.min_lockup_time_secs <= locked_until_secs,
+            Errors::invalid_argument(ELOCK_TIME_TOO_SHORT),
+        );
+        assert!(
+            locked_until_secs <= current_time + validator_set_config.max_lockup_time_secs,
+            Errors::invalid_argument(ELOCK_TIME_TOO_LONG),
+        );
```

Comment:
Nit, it's a little confusing since you're using `locked_until` and `lockup_time` as interchangable between a duration and a time.

I'd name `min/max_lockup_duration_secs` to be more straightforward that it's a diff between now to a time.

Reply:
Updated.